config/routes.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.routes.draw do
```

Comment:
Whats this file about?

Reply:
This is the config file for the Rails router where we can set which files we want to serve when a user hits a particular path in browser url. For instance, we can set that hitting the `/` path will render the `home` template, whereas, hitting the `dashboard` will render the `dashboard` template.

Reply:
close. But incorrect

Reply:
This is the config file for the Rails router where we can set which controller's action to dispatch to when a user hits a particular endpoint in browser URL. In the action controller, we can then specify which view the user should see.

So say a user goes to a site `https://example.com/about`, it looks for a [GET] /about route. If we had specified in the route to go to the `About` controller with an `index` action, it then goes to the `About` controller and find the index action which will then render out whichever html template that was specified.